variables.tf

```diff
@@ -511,23 +511,17 @@ variable "create_static_kubeconfig" {
   default     = true
 }
 
-variable "infra_mode" {
-  description = "Use private IP address for cluster API endpoint"
+variable "cluster_api_mode" {
+  description = "Use Public or Private IP address for the cluster API endpoint"
   type        = string
-  default     = "standard"
+  default     = "public"
 
   validation {
-    condition     = contains(["standard", "private"], lower(var.infra_mode))
-    error_message = "ERROR: Supported values for `infra_mode` are standard, private."
+    condition     = contains(["public", "private"], lower(var.cluster_api_mode))
+    error_message = "ERROR: Supported values for `cluster_api_mode` are - public, private."
   }
 }
 
-variable "vpc_private_endpoints" {
-   description = "Endpoints needed for private cluster"
-   type        = list(string)
-   default     = [ "ec2", "ecr.api", "ecr.dkr", "s3", "logs", "sts", "elasticloadbalancing", "autoscaling" ]
-}
-
```

Comment:
All of these items, as noted above, were in place to keep the infra private internally to help with security. Need to know what/how these items are now handled and locked down as stated above.